tests/integration/llm/run_max_num_tokens.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+import json
+import os
+import urllib.request
+from tensorrt_llm_toolkit.utils.utils import max_token_finder
+
+if __name__ == '__main__':
+    os.mkdir("max_num_token_results")
+
+    parser = argparse.ArgumentParser(description='Parse inputs to script')
+    parser.add_argument('i_model_tp_json',
+                        help='Input - JSON containing model and tp pairs')
+    args = parser.parse_args()
+
+    data = urllib.request.urlopen(args.i_model_tp_json)
+    json_str = [line.decode("utf-8").strip() for line in data][0]
+    model_tp_dict = json.loads(json_str)
+
+    log_id = 0
+    for model_id, tp_list in model_tp_dict.items():
```

Comment:
let's have more log lines print out here

Reply:
Done (both for each model and for each tp)